dot/build_spec_integration_test.go

```diff
@@ -1,8 +1,6 @@
 // Copyright 2021 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
-//go:build integration
```

Comment:
Why is this gone? Unless something has changed we should be running our integration tests as integration tests

Reply:
Uh-oh

Reply:
added it back in